integration-tests/src/test/resources/getT3Client.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Copyright 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+
+SCRIPTPATH="$( cd "$(dirname "$0")" > /dev/null 2>&1 ; pwd -P )"
+
+id=$(docker create store/oracle/weblogic:19.1.0.0)
```

Comment:
can you explain what does docker create do here? also can this be moved to setupenv.sh?

Reply:
It creates a throwaway container to copy the wlthint3client.jar from it. I think it can be moved to setupenv.sh, I created a new script to not pollute other scripts.

Reply:
Moved that script to setupenv.sh